firmware/STM32L4R5ZITx_FLASH.ld

```diff
@@ -64,7 +64,9 @@ MEMORY
 RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 640K
 RAM2 (xrw)      : ORIGIN = 0x10000000, LENGTH = 64K
 RAM3 (xrw)      : ORIGIN = 0x20040000, LENGTH = 384K
-FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 2048K
+FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 1024K
```

Comment:
Nice work figuring this out! 

The IOC code generation overwrites this file. I wonder if there's a way to configure this in the IOC file? Otherwise, it must be added as a `docs/Non-Critical_Notes/Deployment_Final_Checks.md` item to ensure this file has remained not-regenerated.

Reply:
Not sure if you can do it using the IOC file, but I will add it in the docs in a bit

Reply:
It is still not in the docs. Please add to the docs, if it is necessary.